spec/features/my_queue_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+feature 'Video queue' do
+
+  given(:user) { Fabricate(:user) }
+  given!(:category) { Fabricate(:category, name: 'Blockbusters')}
+  given!(:alien) { Fabricate(:video, title: 'Alien', category: category) }
+  given!(:taxi_driver) { Fabricate(:video, title: 'Taxi driver', category: category) }
+
+  background 'logged-in user' do
+    visit sign_in_path
+    fill_in :email, with: user.email
+    fill_in :password, with: user.password
+    click_button 'Sign in'
+  end
```

Comment:
okay I created a module capybara_helper.rb (don't know how to name it better because actually it contains capybara specific syntax) with a sign_in method included in spec_helper.rb.